fluentd/Gemfile.lock

```diff
@@ -59,6 +59,8 @@ GEM
       fluentd (>= 0.10.58, < 2)
```

Comment:
I think this should be replaced by the fluentd requirement below.
```suggestion
      fluentd (>= 0.14.10, < 2)      
```

Reply:
Thanks. I will fix it. 
Just confirmation. fluent-plugin-label-router requires ">= 0.14.10". So minimum version for other components should be also raised. Is my understanding correct? 
If yes, do I need to raise the lower version of other components(0.14.0 and 0.14.2)?
Or is it no problem if 0.14.x is same for the compatibility of fluentd?

Reply:
I updated my patch. If there is no problem, please merge it.